java/Anton-Kozyk/src/com/zoolatech/lecture1/tasks/_7/Task7.java

```diff
@@ -0,0 +1,35 @@
+package com.zoolatech.lecture1.tasks._7;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that prints all odd numbers from N to 1. N should be read from the input.
+ * Every number should be printed on the new line.
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        printOdd(scanner.nextInt());
+        System.out.println();
+        printOdd_2(scanner.nextInt());
+    }
+
+    public static void printOdd(int N) {
+        if (N % 2 == 0) N -= 1;
```

Comment:
note, that you could use another operator instead of `N -= 1` that has similar functions